test/functional/test_framework/messages.py

```diff
@@ -218,6 +219,7 @@ def deserialize(self, f, with_time=True):
     def serialize(self, with_time=True):
```

Comment:
style-nit: Passing pod literals without a name makes code harder to read. My preference would be to rule out the possibility to pass without name.

```suggestion
    def serialize(self, *, with_time=True):
```

Same for unserialize, see https://github.com/bitcoin/bitcoin/pull/14033#issuecomment-416224661

Reply:
Also makes review easier because running the tests proves that no other place than msg_version sets it to false

Reply:
What's a pod literal?

Reply:
An integral value like `True` or `1`. Not sure what the right term for that is

Reply:
I've taken your suggested change